src/node-lib/curriculum-api-2023/queries/unitListing/units/units.schema.ts

```diff
@@ -28,6 +28,9 @@ const unitData = z.object({
   yearOrder: z.number(),
   cohort: z.string().nullish(),
   learningThemes: z.array(learningThemesSchema).nullable(),
+  subjectCategories: z
+    .array(z.object({ label: z.string(), slug: z.string() }).or(z.null()))
```

Comment:
does this mean there could be null values in the subcategory array?

Reply:
Removed additional null 👍